lib/react_on_rails/server_rendering_pool.rb

```diff
@@ -89,13 +41,67 @@ def self.server_render_js_with_console_logging(js_code)
         if console_script_lines
           console_script_lines.each do |line|
             match = re.match(line)
-            if match
-              Rails.logger.info { "[react_on_rails] #{match[:msg]}" }
-            end
+            Rails.logger.info { "[react_on_rails] #{match[:msg]}" } if match
           end
         end
       end
       result
     end
+
+    class << self
```

Comment:
This should make the module methods private. At least it does it on a class. @samnang correct?